share/sysbuild/image_configurations/MAIN_image_default.cmake

```diff
@@ -5,15 +5,16 @@
 # This sysbuild CMake file sets the sysbuild controlled settings as properties
 # on the main Zephyr image.
 
-set_config_bool(${ZCMAKE_APPLICATION} CONFIG_BOOTLOADER_MCUBOOT "${SB_CONFIG_BOOTLOADER_MCUBOOT}")
-set_config_string(${ZCMAKE_APPLICATION} CONFIG_MCUBOOT_SIGNATURE_KEY_FILE
-                  "${SB_CONFIG_BOOT_SIGNATURE_KEY_FILE}"
-)
-set_config_string(${ZCMAKE_APPLICATION} CONFIG_MCUBOOT_ENCRYPTION_KEY_FILE
-                  "${SB_CONFIG_BOOT_ENCRYPTION_KEY_FILE}"
-)
+set_config_bool(${ZCMAKE_APPLICATION} CONFIG_BOOTLOADER_MCUBOOT y)
```

Comment:
this is not correct

Reply:
Changed back to `set_config_bool(${ZCMAKE_APPLICATION} CONFIG_BOOTLOADER_MCUBOOT "${SB_CONFIG_BOOTLOADER_MCUBOOT}")`, it should be correct now.